src/Game/InventoryList.js

```diff
@@ -0,0 +1,67 @@
+import { supabase } from "../supabase_client";
+import { useEffect, useState } from "react";
+
+const InventoryList = (props) => {
+  const { myPlayerData } = props;
+  const [playerInventory, setplayerInventory] = useState([]);
+  const [newCollectedCardID, setNewCollectedCardID] = useState(0);
```

Comment:
Is there a reason you use 0 here? It seems like `undefined` (default if you put nothing) or `null` would be more appropriate

Reply:
Since the card ID values are integers, I just chose 0 as the default starting integer. But null or undefined accomplish the same thing (of determining that no card has been collected yet)